src/Main.java

```diff
@@ -1,5 +1,21 @@
+import exception.ArrayDataException;
+import exception.ArraySizeException;
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        String[][] matrix = {{"1", "2", "3", "4"},
+                             {"1", "2", "3", "4"},
+                             {"1", "2", "3", "4"},
+                             {"1", "2", "3", "4"}};
+
+        ArrayValueCalculator arrayValueCalculator = new ArrayValueCalculator();
+        try {
+            System.out.println(arrayValueCalculator.doCalc(matrix));
+
+        } catch (ArraySizeException arraySizeException) {
```

Comment:
remove `try- catch` from the `main`. According to the task description you were asked to handle all the exceptions in method `doCalc`.